web/frontend/src/components/slam-2d-render.vue

```diff
@@ -192,6 +326,28 @@ onUnmounted(() => {
   disposeScene();
 });
 
+watch(() => [props.destVector!.z, props.destVector!.x, props.destExists], async () => {
+  if (props.destVector && props.destExists) {
+    const marker = scene.getObjectByName('DestinationMarker') ?? await makeMarker(destMarkerUrl, 'DestinationMarker', 0.1);
+    marker.position.set(props.destVector.x + destinationMarkerOffset.x, 0, props.destVector.z + destinationMarkerOffset.z);
```

Comment:
Can you please add `destinationMarkerOffset.y` & provide it here. It's fine if the offset is 0. Trying to keep magic numbers out of function bodies.